src/main/java/no/nav/veilarbvedtaksstotte/client/veilarboppfolging/VeilarboppfolgingClientImpl.java

```diff
@@ -13,45 +13,57 @@
 import org.springframework.cache.annotation.Cacheable;
 import org.springframework.http.HttpHeaders;
 
+import java.util.List;
+import java.util.function.Supplier;
+
+import static no.nav.common.utils.AuthUtils.bearerToken;
 import static no.nav.common.utils.UrlUtils.joinPaths;
-import static no.nav.veilarbvedtaksstotte.utils.RestClientUtils.authHeaderMedInnloggetBruker;
 
 public class VeilarboppfolgingClientImpl implements VeilarboppfolgingClient {
 
     private final String veilarboppfolgingUrl;
 
     private final OkHttpClient client;
 
-    public VeilarboppfolgingClientImpl(String veilarboppfolgingUrl) {
+    private final Supplier<String> userTokenSupplier;
+
+    private final Supplier<String> systemTokenSupplier;
+
+    public VeilarboppfolgingClientImpl(String veilarboppfolgingUrl, Supplier<String> userTokenSupplier, Supplier<String> systemTokenSupplier) {
         this.veilarboppfolgingUrl = veilarboppfolgingUrl;
         this.client = RestClient.baseClient();
+        this.userTokenSupplier = userTokenSupplier;
+        this.systemTokenSupplier = systemTokenSupplier;
     }
 
+    @Cacheable(CacheConfig.OPPFOLGING_CACHE_NAME)
     @SneakyThrows
-    public String hentServicegruppe(String fnr) {
+    public OppfolgingDTO hentOppfolgingData(String fnr) {
         Request request = new Request.Builder()
-                .url(joinPaths(veilarboppfolgingUrl, "/api/person/", fnr, "/oppfolgingsstatus"))
-                .header(HttpHeaders.AUTHORIZATION, authHeaderMedInnloggetBruker())
+                .url(joinPaths(veilarboppfolgingUrl, "/api/oppfolging?fnr=" + fnr))
```

Comment:
Endring til /oppfolging endepunktet er "tyngre", er det nødvendig?

Reply:
Sant det. Så denne kommentaren i veilarboppfolging `/*
     API used by veilarbmaofs. Contains only the necessary information
     */`, så jeg tenkte det kunne vært greit hvis færre bruker den. Gjør det litt enklere å fikse på den senere